core/tests/widgets/canvas/test_canvas.py

```diff
@@ -153,7 +153,7 @@ def test_measure_text(widget, font, expected):
     assert widget.measure_text("Hello world", font=font) == expected
 
 
-def test_as_image(widget):
```

Comment:
As noted in the dummy Canvas - this change won't be needed if you make the resource local to dummy.